src/main/webapp/app/exercises/quiz/manage/re-evaluate/multiple-choice-question/re-evaluate-multiple-choice-question.component.scss

```diff
@@ -1,29 +1,85 @@
 .edit-mc-question {
+    .card-header {
+        padding: 15px 5px 5px 5px;
```

Comment:
Maybe you could change spacings to multiples of 1rem, so that they correspond to bootstrap spacings?

Reply:
Set this to 0 instead and add bootstrap margins to elements inside the card-header.